data/data-pipeline/data_pipeline/etl/score/schemas/datasets.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List, Optional
+
+
+class FieldType(Enum):
+    STRING = "string"
+    INT64 = "int64"
+    BOOL = "bool"
+    FLOAT = "float"
+    PERCENTAGE = "percentage"
+    LOSS_RATE_PERCENTAGE = "loss_rate_percentage"
+
+
+@dataclass
+class DatasetsConfig:
+    @dataclass
+    class Dataset:
+        """A class that defines a dataset and its load variables.
+
+        Attributes:
+            long_name (str): A human readable title for the dataset.
+            short_name (str): used to compose the short variable names for tiles/arcgis. All short variable names will be prepended
+            with the short name of the data set it comes from, i.e. `nri__ex_loss`.
+            module_name (str): A string that matches both the Python module name for the dataset and the `NAME` property on the ETL class.
+            description (str): A human readable description of the dataset.
```

Comment:
I think we'll actually want two descriptions, sorry for not catching that sooner. Maybe `description_short` and `description_long`? 

One is the description that shows up in the sidebar. E.g., for field "Expected agriculture loss rate", the `description_short = Economic loss rate to agricultural value resulting from natural hazards each year.` 

<img width="265" alt="Screen Shot 2022-07-06 at 6 59 35 PM" src="https://user-images.githubusercontent.com/25443529/177657012-bd2fe3be-f229-491f-866c-cf13aede8506.png">

The other description is on the data and methodology page. E.g., for field "Expected agriculture loss rate", the `description_long = Percent of agricultural value at risk from losses due to fourteen types of natural hazards that have some link to climate change: avalanche, coastal flooding, cold wave, drought, hail, heat wave, hurricane, ice storm, landslide, riverine flooding, strong wind, tornado, wildfire, and winter weather. Rate calculated by dividing the agricultural value at risk in a census tract by the total agricultural value in that census tract.` 

<img width="499" alt="Screen Shot 2022-07-06 at 7 01 02 PM" src="https://user-images.githubusercontent.com/25443529/177657141-ec820036-37f9-4871-810f-8d9a7b7ce0fe.png">

Reply:
Agreed! Done!